include/wx/private/uilocale.h

```diff
@@ -12,6 +12,15 @@
 
 #include "wx/localedefs.h"
 #include "wx/string.h"
+#include "wx/vector.h"
+
+typedef wxVector<wxLanguageInfo> wxLanguageInfos;
+
+// Return the vector of all known languages.
+const wxLanguageInfos& wxGetLanguageInfos();
```

Comment:
(Beware that the included database is incomplete and doesn't even cover all languages supported by e.g. modern Windows, so "all" is a bit of a stretch.)

Reply:
It was supposed to be "known to wx", I'll update it.